command/cliconfig/cliconfig.go

```diff
@@ -136,6 +153,13 @@ func loadConfigFile(path string) (*Config, tfdiags.Diagnostics) {
 		return result, diags
 	}
 
+	// Deal with the provider_installation block, which is not handled using
+	// DecodeObject because its structure is not compatible with the
```

Comment:
I started to write a comment that I'd like to see more detail here for future readers (ie, me) but found it in the function instead so 🎉 thank you!